src/docs/asciidoc/core/core-aop.adoc

```diff
@@ -545,14 +545,14 @@ this purpose. A typical such aspect would look as follows:
 		 *
 		 * If you group service interfaces by functional area (for example,
 		 * in packages com.xyz.someapp.abc.service and com.xyz.someapp.def.service) then
-		 * the pointcut expression "execution(* com.xyz.someapp..service.*.*(..))"
+		 * the pointcut expression "execution(* com.xyz.someapp.service.*.*(..))"
```

Comment:
Without looking too deep into it, I believe the "fix" here is wrong as using two-dots is actually intended in order to match both the `abc` and `def` package.

Reply:
You're right. I just amended the commit.